README.rst

```diff
@@ -16,11 +16,11 @@ pre-processing implementations, resulting in challenges such as
 portability of training and inference workflows, and code
 maintainability.
 
-NVIDIA Data Loading Library (DALI) is a collection of highly optimized
+NVIDIA DALI (R) (Data Loading Library) is a collection of highly optimized
```

Comment:
```suggestion
NVIDIA DALI (R) (Data Loading Library) is a collection of highly optimized
```
1. That's not how we call it. The full name is "NVIDIA Data Loading Library"
2. NVIDIA may be (R), but DALI is (TM) at most - though I doubt it's any of the above.
I'd revert it to the original form.

Reply:
https://trademarks.justia.com/878/24/nvidia-87824723.html
So TM in that case